src/aiida/storage/sqlite_dos/backend.py

```diff
@@ -45,7 +50,19 @@
 FILENAME_CONTAINER = 'container'
 
 
+ALEMBIC_REL_PATH = 'migrations'
+
+REPOSITORY_UUID_KEY = 'repository|uuid'
+
+
 class SqliteDosMigrator(PsqlDosMigrator):
+    """Class for validating and migrating `sqlite_dos` storage instances.
+
+    .. important:: This class should only be accessed via the storage backend class (apart from for test purposes)
```

Comment:
It would be nice to somewhere include a high-level description of how is the SqliteDosMigrator different from its base implementation (based off PsqlDosMigrator) i.e. what you have written up on the PR.

Reply:
It is not so much that the mechanism is different, it is just that it provides a separate set of actual migrations, which will have to be sqlite or psql specific since they can use direct SQL features that are only supported by one or the other. Before, all plugins used the same migrations which were written for PSQL and so failed for sqlite databases.

Only other difference is that the new sqlite-migrator does not have the complicated additional logic to deal with migrating from legacy Django databases as this was introduced _after_ Django had already been dropped and there were just sqlalchemy databases. I will add something to the docstring.